### PR TITLE
fix(Customizer/General_Theme) get value if set

### PR DESCRIPTION
**Pull request by lucatume**
Issue: n/a

This PR fixes an issue where a Customizers setting value would be fetched without checking if it's set.

Related to: https://github.com/the-events-calendar/tribe-common/pull/1619 and https://app.travis-ci.com/github/the-events-calendar/tribe-common/builds/232502340#L2084

### CODE REVIEW

File: src/Tribe/Customizer/General_Theme.php [hunk 1]
```diff
@@ -33,7 +33,9 @@ public static function instance() {
 	 * @return string
 	 */
 	protected function get_button_bg_color( $settings = [] ) {
-		$scheme = $this->sanitize_featured_color_choice( $settings['featured_color_scheme'] );
+		$scheme = isset( $settings['featured_color_scheme'] )
```
Comment (Camwyn) [inline src/Tribe/Customizer/General_Theme.php:36]: This works, but we should also just not call this if it's not set?

Comment (Camwyn) [reply src/Tribe/Customizer/General_Theme.php:36]: I pushed a tweak.

**Review by Camwyn — APPROVED** [review]
Approving but I added code.

**Review by borkweb — DISMISSED** [review]
Tests appear to be failing

**Pull request merged**